lib/hanami/utils/kernel.rb

```diff
@@ -404,15 +413,14 @@ def self.Integer(arg)
       #   # Missing #respond_to?
       #   input = BasicObject.new
       #   Hanami::Utils::Kernel.BigDecimal(input) # => TypeError
-      def self.BigDecimal(arg)
+      def self.BigDecimal(arg, precision: BIGDECIMAL_DEFAULT_PRECISION)
```

Comment:
@pascalbetz Interesting, thank you. What about to measure the new implementation at https://github.com/hanami/utils/commit/9fb7bcf036e64d7180539c2e2b4b6e2d37e739cb and eventually refactor with `if/else`? Thank you very much :+1: